packages/cfworkers/src/mod.test.ts

```diff
@@ -0,0 +1,79 @@
+import type {
+  KVNamespace,
+  Queue,
+} from "@cloudflare/workers-types/experimental";
+import { assertEquals } from "@std/assert";
+import { delay } from "es-toolkit";
+import { WorkersKvStore, WorkersMessageQueue } from "./mod.ts";
+
+Deno.test({
+  name: "WorkersKvStore",
+  ignore: !("navigator" in globalThis &&
+    navigator.userAgent === "Cloudflare-Workers"),
+  async fn(t) {
+    const { env } = t as unknown as {
+      env: Record<string, KVNamespace<string>>;
+    };
+    const store = new WorkersKvStore(env.KV1);
+
+    await t.step("set() & get()", async () => {
+      await store.set(["foo", "bar"], { foo: 1, bar: 2 });
+      assertEquals(await store.get(["foo", "bar"]), { foo: 1, bar: 2 });
+      assertEquals(await store.get(["foo"]), undefined);
+
+      await store.set(["foo", "baz"], "baz", {
+        ttl: Temporal.Duration.from({ seconds: 0 }),
+      });
+      assertEquals(await store.get(["foo", "baz"]), undefined);
+    });
+
+    await t.step("delete()", async () => {
+      await store.delete(["foo", "bar"]);
+      assertEquals(await store.get(["foo", "bar"]), undefined);
+    });
+  },
+});
+
+Deno.test({
```

Comment:
In order to make this test runnable on Node.js and Bun, we should use [`node:test`](https://nodejs.org/api/test.html) and [`node:assert/strict`](https://nodejs.org/api/assert.html).